actions.yaml

```diff
@@ -11,3 +11,11 @@ rotate-wordpress-secrets:
     auth_key, auth_salt, logged_in_key, logged_in_salt, nonce_key, nonce_salt, secure_auth_key,
     secure_auth_salt.
     Users will be forced to log in again. This might be useful under security breach circumstances.
+update-database:
+  description: >
+    After upgrading WordPress to a new version it is typically necessary to run 'wp core update-db'
+    to migrate the database schema. This action does exactly that.
+  params:
+    dry-run:
+      type: boolean
+      description: Runs the 'wp core update-db --dry-run' command.
```

Comment:
nit: final new line missing

Reply:
It doesn't have a final new line in the original version. Also should all yaml files have an empty last line?

Reply:
Its like an unwritten convention in our team (afaik). But if it was not there before, you can ignore my comment and close this conversation.

Reply:
Please do add a new-line :D Reason: https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3